packages/pilot/src/containers/AddRecipient/ConfigurationStep/index.js

```diff
@@ -199,6 +199,7 @@ class ConfigurationsStep extends Component {
           </CardActions>
         </Form>
         <HelpModal
+          anticipationModel={data.anticipationModel}
```

Comment:
isso é um typo? se for, tá reproduzindo nos outros arquivos também...
```suggestion
          anticipationModal={data.anticipationModal}
```